snomed/com.b2international.snowowl.snomed.importer/src/com/b2international/snowowl/snomed/importer/net4j/ImportConfiguration.java

```diff
@@ -18,13 +18,16 @@
 import java.io.File;
```

Comment:
I get the following exception when I open the import wizard in the client (server-client mode): 
`java.lang.IllegalArgumentException: Could not find property with name descriptionsFile in class class com.b2international.snowowl.snomed.importer.net4j.ImportConfiguration
	at org.eclipse.core.internal.databinding.beans.BeanPropertyHelper.getPropertyDescriptor(BeanPropertyHelper.java:168)
	at org.eclipse.core.databinding.beans.PojoProperties.value(PojoProperties.java:119)
	at org.eclipse.core.databinding.beans.PojoProperties.value(PojoProperties.java:89)
	at org.eclipse.core.databinding.beans.PojoObservables.observeValue(PojoObservables.java:76)
	at org.eclipse.core.databinding.beans.PojoObservables.observeValue(PojoObservables.java:56)
	at com.b2international.snowowl.snomed.importer.ui.wizard.page.ImportDetailsWizardPage.bindTextField(ImportDetailsWizardPage.java:319)
	at com.b2international.snowowl.snomed.importer.ui.wizard.page.ImportDetailsWizardPage.initDataBinding(ImportDetailsWizardPage.java:236)
	at com.b2international.snowowl.snomed.importer.ui.wizard.page.AbstractImportWizardPage.createControl(AbstractImportWizardPage.java:67)`